runtime/testnet/src/config/api/mod.rs

```diff
@@ -99,24 +99,27 @@ parameter_types! {
 
 impl messaging::Config for Runtime {
 	type CallbackExecutor = CallbackExecutor;
+	// Burn fees.
+	type FeeHandler = ();
 	type Fungibles = Balances;
 	type IsmpDispatcher = Ismp;
 	type IsmpRelayerFee = IsmpRelayerFee;
+	type Keccak256 = Ismp;
 	type MaxContextLen = ConstU32<64>;
-	type MaxDataLen = ConstU32<1024>;
-	type MaxKeyLen = ConstU32<32>;
+	type MaxDataLen = ConstU32<512>;
+	type MaxKeyLen = ConstU32<8>;
```

Comment:
I believe this will be a problem. Look at this storage key for asset 1 in asset hub: `0x682a59d51ab9e48a8c8cc418ff9708d2d34371a193a751eea5883e9553457b2ed82c12285b5d4551f88e8f6e7eb52b8101000000`

The length is well above 8. Something like 64 or 128 might be more appropriate..

Reply:
i will create a task to reason about this issue. Cases like this are really useful.

Reply:
ref: https://app.shortcut.com/r0gue/story/3704/proper-parameters-for-ismp-keys-config